integrations/influxdb/influxdb-metrics/sensu-resources.yaml

```diff
@@ -27,54 +27,56 @@ spec:
   command: >-
     sensu-influxdb-handler
     --addr ${INFLUXDB_ADDR}
-    --db-name ${INFLUXDB_DB}
+    --org ${INFLUXDB_ORG}
+    --bucket ${INFLUXDB_BUCKET}
   runtime_assets:
-    - sensu/sensu-influxdb-handler:3.7.0
+    - sensu/sensu-influxdb-handler:4.0.0
   env_vars: []
   secrets: []
   timeout: 10
 ---
 type: Asset
 api_version: core/v2
 metadata:
-  name: sensu/sensu-influxdb-handler:3.7.0
+  name: sensu-influxdb-handler:4.0.0
```

Comment:
This should be `sensu/sensu-influxdb-handler:4.0.0` I think. Will edit to reflect that.